packages/Modal/Header.tsx

```diff
@@ -5,8 +5,8 @@ import { forwardRef } from '@welcome-ui/system'
 import { useTheme } from '@xstyled/styled-components'
 
 export interface HeaderOptions {
-  title: string
-  subtitle?: string
+  title: string | JSX.Element
+  subtitle?: string | JSX.Element
```

Comment:
i think React.ReacNode is the good one in this case
https://stackoverflow.com/a/58123882

Reply:
On fait quoi alors @guillaumewttj @mleralec ? Quelle est la solution qu'on aimerait mettre partout ?

Reply:
Les deux sont pertinents ici. `JSX.Element` est peut-être plus permissif (peut-être mieux dans une lib ?)

Reply:
So @guillaumewttj @mleralec what do you decide? 🤣

Reply:
Si pas de décision on va merger comme cela ;)

Reply:
Ok, I merge.